tests/auto/auto.pro

```diff
@@ -8,6 +8,7 @@ SUBDIRS = \
         logging \
         loggingdirect \
         loggingloading \
+        macaddressunittests \
```

Comment:
would follow the naming convention, that is, without "unittests" and the binary named "testmacaddress"

Reply:
Added to make clear this are unittests, the other tests are integration tests...